README.md

```diff
@@ -31,14 +31,33 @@ This page id serves as the canonical identifier of the page, and is used in othe
 
 The output map is a tsv file that looks like this (example from Turkish wiki dump for 20181020):
 
+````
 10	Cengiz_Han	0
 16	Film_(anlam_ayrımı)	0
 22	Mustafa_Suphi	0
 24	Linux	0
 25	MHP	1
+````
 
 Each line represents an entry for one page, where the first field is the page id, the second field is the page title, and the third field is a boolean indicating whether the page is redirection.
 
+### Wikipedia Page hyperlink json output
```

Comment:
The user should not need to add this to the makefile. Simply modify the makefile and add this. 
The user should just need to specify something like `lang=zh make links` to get the json output.

Reply:
Hi, I already added this to makefile, just write here to note the change.